.github/workflows/python.yml

```diff
@@ -0,0 +1,92 @@
+name: CI
+
+on: [push]
```

Comment:
Shouldn't the GitHub action run on both push and pull requests?

Reply:
Fixed please check now